include/depthai/depthai_wrapper.hpp

```diff
@@ -0,0 +1,57 @@
+#include "depthai/device.hpp"
+#include <string>
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <fstream>
+#include <sstream>
+#include "nlohmann/json.hpp"
+#include <vector>
+
+namespace DepthAI
+{
+
+using CV_mat_ptr = std::shared_ptr<cv::Mat>;  // shared ptr for cv::Mat
+
+class DepthAI : public Device
+{
+
+public:
+    /**  Constructor:
+     *   initializing OAK device and creating pipeline
+     *   to capture camera streams and AI outputs.
+     */
+    DepthAI(std::string usb_device, std::string config_file, bool usb2_mode = false);
+
+    /* API to stream output frames from OAK.
+     * AI data and metadata is WIP
+     */
+
+    void get_frames(std::unordered_map<std::string, CV_mat_ptr>& output_streams);
+
+    /* Destructor */
+    ~DepthAI();
+
+private:
+    using PacketsTuple = std::tuple<
```

Comment:
Maybe you should also move it to the top of the file with the other `using` tag.